release/models/devices-manifest/openconfig-terminal-device-property-types.yang

```diff
@@ -0,0 +1,459 @@
+    base MODULATION_FORMAT;
+    description
+      "Dual-Carrier Dual-Polarization 8-Quadrature Amplitude
+      Modulation identity";
+  }
+
+  identity MODULATION_FORMAT_16QAM {
+    base MODULATION_FORMAT;
+    description
+      "16-Quadrature Amplitude Modulation (QAM) identity";
+  }
+
+  identity MODULATION_FORMAT_DP_16QAM {
+    base MODULATION_FORMAT;
+    description
+      "Dual-Polarization 16-Quadrature Amplitude Modulation (QAM)
+      identity";
+  }
+
+  identity MODULATION_FORMAT_DC_DP_16QAM {
+    base MODULATION_FORMAT;
+    description
+      "Dual-Carrier Dual-Polarization 16-Quadrature Amplitude Modulation
+      (QAM) identity";
+  }
+
+  identity MODULATION_FORMAT_32QAM {
+    base MODULATION_FORMAT;
+    description
+      "32-Quadrature Amplitude Modulation (QAM) identity";
+  }
+
+  identity MODULATION_FORMAT_DP_32QAM {
+    base MODULATION_FORMAT;
+    description
+      "Dual-Polarization 32-Quadrature Amplitude Modulation (QAM)
+      identity";
+  }
+
+  identity MODULATION_FORMAT_DC_DP_32QAM {
+    base MODULATION_FORMAT;
+    description
+      "Dual-Carrier Dual-Polarization 32-Quadrature Amplitude Modulation
+      (QAM) identity";
+  }
+
+  identity MODULATION_FORMAT_64QAM {
+    base MODULATION_FORMAT;
+    description
+      "64-Quadrature Amplitude Modulation (QAM) identity";
+  }
+
+  identity MODULATION_FORMAT_DP_64QAM {
+    base MODULATION_FORMAT;
+    description
+      "Dual-Polarization 64-Quadrature Amplitude Modulation (QAM)
+      identity";
+  }
+
+  identity MODULATION_FORMAT_DC_DP_64QAM {
+    base MODULATION_FORMAT;
+    description
+      "Dual-Carrier Dual-Polarization 64-Quadrature Amplitude Modulation
+      (QAM) identity";
+  }
+
+  identity MODULATION_FORMAT_PAM4 {
+    base MODULATION_FORMAT;
+    description
+      "4-level Pulse Amplitud Modulation (PAM)";
+  }
+
+  identity MODULATION_FORMAT_PAM8 {
+    base MODULATION_FORMAT;
+    description
+      "8-level Pulse Amplitud Modulation (PAM)";
+  }
+
+  identity MODULATION_FORMAT_PCS {
+    base MODULATION_FORMAT;
+    description
+      "Probabilistic Constellation Shaping.";
+  }
+
+  identity TRANSCEIVER_MODE_TYPE{
+    description
+      "Base identity for identiying the transceiver's mode category:
+      Standard or Explicit. This differentiates on the way operational
+      mode's features are expose by the transceiver device.";
+  }
+
+  identity TRANSCEIVER_MODE_TYPE_STANDARD{
+    base TRANSCEIVER_MODE_TYPE;
+    description
+      "Operational model type Standard, according to ITU-T G.698.2,
+      indicates the mode is interoperable with other transceiver-modules
+      supporting the same Standard mode.";
+  }
+
+  identity TRANSCEIVER_MODE_TYPE_EXPLICIT{
+    base TRANSCEIVER_MODE_TYPE;
+    description
+      "Operational model type Explicit, indicating the description of the
+      operational mode features follows the model subtree explicit
+      features description.";
+  }
+
+
+}
```

Comment:
I think prefers a newline at the end of the file.

Reply:
Included in the latest commit.